.credo.exs

```diff
@@ -51,107 +64,154 @@
       #
       #     {Credo.Check.Design.DuplicatedCode, false}
       #
-      checks: [
-        #
-        ## Consistency Checks
-        #
-        {Credo.Check.Consistency.ExceptionNames, []},
-        {Credo.Check.Consistency.LineEndings, []},
-        {Credo.Check.Consistency.MultiAliasImportRequireUse, false},
-        {Credo.Check.Consistency.ParameterPatternMatching, []},
-        {Credo.Check.Consistency.SpaceAroundOperators, false},
-        {Credo.Check.Consistency.SpaceInParentheses, false},
-        {Credo.Check.Consistency.TabsOrSpaces, []},
-        {Credo.Check.Consistency.UnusedVariableNames, false},
+      checks: %{
+        enabled: [
+          #
+          ## Consistency Checks
+          #
+          {Credo.Check.Consistency.ExceptionNames, []},
+          {Credo.Check.Consistency.LineEndings, []},
+          {Credo.Check.Consistency.ParameterPatternMatching, []},
+          {Credo.Check.Consistency.SpaceAroundOperators, []},
```

Comment:
I think you actually altered the configured checks when you updated the config, do you mind ensuring the same checks enabled as before?

Reply:
Updated 👍